src/main/webapp/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="UTF-8">
+<title>cfe_18</title>
+<style>
+.circle
```

Comment:
create ticket for ux designer to work on this

Reply:
Done